### PR TITLE
Fix of name for one group and speed improvement

### PR DESCRIPTION
**Pull request by hulpke**
Fix the name of PrimitiveGroup(81,125) (looks as if it was a typing mistake when creating)

Improve speed of `PrimitiveIdentification` by testing also the abelian invariants.

### CODE REVIEW

File: data/gps1.g [hunk 1]
```diff
@@ -4135,7 +4135,7 @@ PRIMGRP[81]:=
       [ [ 0*Z(3), 0*Z(3), Z(3)^0, 0*Z(3) ], [ Z(3), Z(3)^0, Z(3)^0, Z(3) ], 
           [ Z(3), 0*Z(3), 0*Z(3), 0*Z(3) ], [ Z(3)^0, Z(3), 0*Z(3), Z(3) ] ] 
      ] ],
-[ 125, 38880, 0, "1", [ [ 40, 2 ] ], 1, "3^5:4.Sym(5)", [ "Z", 3, 4 ], 
+[ 125, 38880, 0, "1", [ [ 40, 2 ] ], 1, "3^4:4.Sym(5)", [ "Z", 3, 4 ],
```
Comment (fingolfin) [inline data/gps1.g]: Oops, good catch

File: lib/primitiv.gi [hunk 2]
```diff
@@ -215,6 +215,18 @@ local dom,deg,PD,s,cand,a,p,s_quot,b,cs,n,beta,alpha,i,ag,bg,q,gl,hom;
...
   fi;
 
+  # AbelianInvariants
+  if Length(cand)>1 then
+    a:= AbelianInvariants(grp);
+    b:= [];
+    for i in [1..Length(cand)] do
+      b[i]:= AbelianInvariants(p[i]);
+    od;
+    s:= Filtered([1..Length(cand)], i->b[i] =a);
```
Comment (fingolfin) [inline lib/primitiv.gi]: Why not just this, and then get rid of the line which init `b` above?
```suggestion
    s:= Filtered([1..Length(cand)], i-> AbelianInvariants(p[i]) =a);
```
Anyway, of course it is also fine this way, just wondering :-)

Comment (hulpke) [reply lib/primitiv.gi]: I was duplicating the structure of the surrounding tests (in the hope that reduces accidental errors and makes the test not stick out)

**Comment by alex-konovalov** [comment]
Nice - thanks @hulpke! We can now publish a new release to be ready for GAP 4.11.

**Pull request merged**